### PR TITLE
Small improvement to homepage.js

### PR DESCRIPTION
**Pull request by valdrinkoshi**
Avoids querying for the same elements multiple times.

### CODE REVIEW

File: assets/js/homepage.js [hunk 1]
```diff
@@ -18,8 +18,9 @@
 			return;
 		}
 
-		for ( var i = 0; i < homepageContent.querySelectorAll( '.entry-title, .entry-content' ).length; i++ ) {
-			homepageContent.querySelectorAll( '.entry-title, .entry-content' )[ i ].classList.add( 'loaded' );
+		var entries = homepageContent.querySelectorAll( '.entry-title, .entry-content' );
+		for ( var i = 0, l = entries.length; i < l; i++ ) {
```
Comment (Aljullu) [inline assets/js/homepage.js:22]: Being honest, I don't think there is a big performance gain assigning `entries.length` to `l`, but it does make the code a bit more verbose. Could we simplify it as so?
```suggestion
		for ( var i = 0; i < entries.length; i++ ) {
```

**Comment by haszari** [comment]
> Thanks for the contribution @valdrinkoshi! Code looks safe and it worked fine in my testing.

Agreed, this is a nice tidy up, don't like seeing the query in the loop body - thanks :)

> will make it sense to add this to 2.5.6?

@Aljullu yes - added to milestone.

File: assets/js/homepage.js (hunk 1 above)
Comment (haszari) [reply assets/js/homepage.js:22]: I agree - also this is unusual style (multiple assign in the `for` statement). More idiomatic and no downside to just using `entries.length` directly as suggested above. Can you make this change please and then we'll merge :)

Comment (valdrinkoshi) [reply assets/js/homepage.js]: Sounds good, done 👌

Comment (haszari) [reply assets/js/homepage.js]: Thank you! Comment when you're done testing and I'll merge :)

Comment (valdrinkoshi) [reply assets/js/homepage.js]: @haszari is that a task for me or @Aljullu?

Comment (haszari) [reply assets/js/homepage.js]: Just confirming you've tested locally @valdrinkoshi :)

File: assets/js/homepage.js [hunk 2]
```diff
@@ -18,8 +18,9 @@
 			return;
 		}
 
-		for ( var i = 0; i < homepageContent.querySelectorAll( '.entry-title, .entry-content' ).length; i++ ) {
-			homepageContent.querySelectorAll( '.entry-title, .entry-content' )[ i ].classList.add( 'loaded' );
+		var entries = homepageContent.querySelectorAll( '.entry-title, .entry-content' );
+		for ( var i = 0, i < entries.length; i++ ) {
```
Comment (Aljullu) [inline assets/js/homepage.js:22]: Thanks for applying the feedback @valdrinkoshi! Just one last comment, I think there is a typo:
```suggestion
		for ( var i = 0; i < entries.length; i++ ) {
```

Comment (valdrinkoshi) [reply assets/js/homepage.js]: Argh, great catch! Fixed 👌

**Pull request merged**